sockshop/front-end/Makefile

```diff
@@ -0,0 +1,37 @@
+.PHONY: test coverage
```

Comment:
Not really. I tried every single task in the Makefile (I wrote it myself) and the only two tasks that have matching folders are these two.